test/test_other.py

```diff
@@ -8131,6 +8131,58 @@ def test_wasm_exceptions_stack_trace_and_message(self):
     for check in stack_trace_checks:
       self.assertNotContained(check, err)
 
+  def test_emscripten_exceptions_stack_trace_and_message(self):
```

Comment:
Rather than adding a whole new test, can we reuse this [`test_wasm_exceptions_stack_trace_and_message`](https://github.com/emscripten-core/emscripten/blob/5402fc921374cf11f19600e5c3f23eeae76fba5c/test/test_other.py#L8066-L8135) by changing its name to just `test_exceptions_stack_trace_and_message` and add using the [`@with_both_eh_sjlj`](https://github.com/emscripten-core/emscripten/blob/5402fc921374cf11f19600e5c3f23eeae76fba5c/test/test_core.py#L91-L124) decorator, like this? https://github.com/emscripten-core/emscripten/blob/5402fc921374cf11f19600e5c3f23eeae76fba5c/test/test_core.py#L1250-L1251

I think that requires moving that decorator to `https://github.com/emscripten-core/emscripten/blob/main/test/common.py` though. Also I'm not 100% sure if that can work with `@requires_v8`, which was added due to https://github.com/emscripten-core/emscripten/blob/5402fc921374cf11f19600e5c3f23eeae76fba5c/test/test_other.py#L8066-L8072

Also this may require changing our stack trace reference strings in the test somewhat. Anyway, I think eventually we should be using a single method for this, but that requires a lot more work than expected, please let me know; I can fix that later.

Reply:
Reused with commit 88963521e76430a54bae6a5cb8a0afe6e2a3992a. :)